src/components/productDetails/directionalButton.css

```diff
@@ -0,0 +1,5 @@
+.is-vertical-centered {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
i really need to pick your brain on this, i'm having a lot of issues for vertical and horizontal alignment